.gitignore

```diff
@@ -39,24 +39,25 @@ Untitled*.ipynb
 # backup files from tools like aspell and sed
 *.bak
 
-# data files
-*.csv
-*.dat
-*.data
-*.db
-*.feather
-*.ndjson
-*.npy
-*.parquet
-*.pkl
-*.pq
-*.rda
-*.Rdata
-*.rds
-*.sqlite
-*.xls
-*.xlsm
-*.xlsx
+## data files
```

Comment:
in my opinion, checking data into the repo should be the exception, not the rule. This repo is cloned every time Saturn starts up, so growing its size substantially will slow down the start up time for the application (which also means increasing the time-to-recovery during a migration / upgrade).

So instead of allowing these types of files into the repo, I'd prefer to make the default behavior "do not allow data in the repo", and have this PR add specific exclusions for the files you want to add.

```shell
*.csv
!examples/dashboard/data/pickup_grouped_by_time.csv
!examples/dashboard/data/pickup_grouped_by_zone.csv
```

If that suggestion is rejected and we decide to go with "allow any data files in the repo", then I'd like to recommend just removing all these gitignore rules instead of commenting them out. Commented-out code doesn't change the behavior of the software in this repo, but it increases the size of the repo and can lead to false positives from search (like `git grep` or GitHub search).

Reply:
conversation is happening in slack about this.

Reply:
oh ok, I see the conversation now. I'll remove my blocking review